test/e2e/subscription_e2e_test.go

```diff
@@ -2488,6 +2537,20 @@ func subscriptionStateUpgradeAvailableChecker(subscription *operatorsv1alpha1.Su
 	return subscription.Status.State == operatorsv1alpha1.SubscriptionStateUpgradeAvailable
 }
 
+func subscriptionHasResolutionFailedCondChecker(subscription *operatorsv1alpha1.Subscription) bool {
+	if cond := subscription.Status.GetCondition(operatorsv1alpha1.SubscriptionResolutionFailed); cond.Status == corev1.ConditionTrue {
+		return true
+	}
+	return false
+}
+
+func subscriptionDoesNotHaveResolutionFailedCondChecker(subscription *operatorsv1alpha1.Subscription) bool {
```

Comment:
looks like this function isn't being used anywhere

Reply:
Removing both of these functions while changing the test according to https://github.com/operator-framework/operator-lifecycle-manager/pull/2269#discussion_r674067049